scss/_tables.scss

```diff
@@ -187,3 +187,20 @@
     }
   }
 }
+
+
+// Fixed header
+//
+// Table header with the `.thead-fixed` class applied fix its cells on top
+// allowing only <tbody> to scroll.
+
+// stylelint-disable-next-line no-duplicate-selectors
+.table {
+  .thead-fixed {
+    th {
+      position: sticky;
```

Comment:
This needs to be wrapped in `@supports (position: sticky) { ... }`

Reply:
Done.